feature-libs/product-configurator/_index.scss

```diff
@@ -1,5 +1,8 @@
 @import '~@spartacus/styles/scss/core';
 
+// TODO: don't commit this line:
+$useLatestStyles: true;
+
```

Comment:
see ToDo ;-)

Reply:
@Larisa-Staroverova: seems like you accidentally commited a line, or the comment is outdated.

Reply:
Actually, this flag should be removed before the epic/cpq-configurator is merged into develop. Until then this flag should be activated, otherwise the "versioned" styling is inactive. Yesterday I created a [task](https://github.com/SAP/spartacus/issues/11238) to update the style version to the write one and remove **useLatestStyles** flag.

Reply:
@Larisa-Staroverova :Can you plz. update the comment accordingly. for example:
//TODO: Please remove this line before releasing as part of task https://github.com/SAP/spartacus/issues/11238